packages/backend-common/src/reading/AzureUrlReader.test.ts

```diff
@@ -116,8 +117,9 @@ describe('AzureUrlReader', () => {
         treeResponseFactory,
       });
 
-      const data = await reader.read(url);
-      const res = await JSON.parse(data.toString('utf-8'));
+      const data = await reader.readUrl(url);
```

Comment:
`readUrl` actually a `ReadUrlResponse` https://github.com/backstage/backstage/blob/master/packages/backend-common/src/reading/types.ts#L121 which contains buffer. So you can do
`const { buffer } = await reader.readUrl(url);` which mean you don't need getRawBody. This goes for the other files as well